packages/rolldown/tests/fixtures/output/fileNames/hash/_config.ts

```diff
@@ -0,0 +1,25 @@
+import type { RolldownOutputChunk } from 'rolldown'
+import { defineTest } from '@tests'
+import { expect } from 'vitest'
+
+export default defineTest({
+  config: {
+    output: {
+      entryFileNames: '[name]-[hash:6].js',
+      chunkFileNames: '[name]-[hash:7].js',
```

Comment:
What if we need more than 8?

Reply:
We seem to support hash lengths of `5-22`.
https://github.com/rolldown/rolldown/blob/e52ef2f22015b919e958bdcf332bfca7425144d2/crates/rolldown/src/utils/hash_placeholder.rs#L44

Reply:
Is [this](https://github.com/rolldown/rolldown/blob/main/crates/rolldown_common/src/file_emitter.rs#L86) `8` related?